src/NLog.Loki/HttpLokiTransport.cs

```diff
@@ -49,19 +49,25 @@ public async Task WriteLogEventsAsync(LokiEvent lokiEvent)
     /// Prepares the HttpContent for the loki event(s).
     /// If gzip compression is enabled, prepares a gzip stream with the appropriate headers.
     /// </summary>
-    private HttpContent CreateContent<T>(T lokiEvent)
+    private async Task<HttpContent> CreateContent<T>(T lokiEvent)
```

Comment:
Create a different, async method for .NET Framework 4.8.
.NET 6+ will remain sync.